patches/server/0738-Expose-tracked-entities.patch

```diff
@@ -0,0 +1,28 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: SuperKooks <superkooks@superkooks.com>
+Date: Wed, 4 Aug 2021 18:05:01 +1000
+Subject: [PATCH] Expose tracked entities
+
+
+diff --git a/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java b/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
+index 9e8918d03b8213e5f6689fc93030138fd704aca9..2a376cb1c4883db02257099ef07c826393ef72d0 100644
+--- a/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
++++ b/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
+@@ -2375,6 +2375,17 @@ public class CraftPlayer extends CraftHumanEntity implements Player {
+     }
+     // Paper end
+ 
++    // Paper start
++    @Override
++    public <T extends org.bukkit.entity.Entity> Collection<? extends T> getTrackedEntities(Class<? extends T> entityClass) {
++        if (entityClass.equals(Player.class)) {
```

Comment:
```suggestion
+        if (entityClass == Player.class) {
```
There should only ever be one instance of a class loaded.

Reply:
:+1: